pages/common/gist.md

```diff
@@ -0,0 +1,23 @@
+# gist
+
+> Upload code to https://gist.github.com.
+
+- Upload any number of files:
+
+`gist {{file.txt}} {{file2.txt}}`
+
+- Upload a private file with a description:
+
+`gist -p {{file.txt}} -d {{"A meaningful description"}}`
+
+- Read contents from STDIN with a name:
```

Comment:
I think "Read contents from STDIN and create a gist from it, with a name:" could be more accurate.